spec/README.md

```diff
@@ -124,6 +122,13 @@ Type: `String`
 
 The associated language written in English.
 
+###### metadata.variant
+
+Optional  
+Type: `String`
+
+The associated language variant written in English, if applicable. For example, if the [ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) code is `de_AT`, this entry would include the full name of the variant country, `"Austria"` for the `AT` variant.
```

Comment:
Please remove `"` around Austria, otherwise some folks might interpret that the value will be `"Austria"`, but the `"` are just necessary in the JSON.